graylog2-web-interface/src/contexts/InputSetupWizardContext.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+
+import { singleton } from 'logic/singleton';
+
+export const INPUT_WIZARD_STEPS = {
+  SELECT_CATEGORY: 'SELECT_CATEGORY',
+  PREFLIGHT: 'PREFLIGHT',
+  ACTIVATE_ILLUMINATE: 'ACTIVATE_ILLUMINATE',
+  TEST_INPUT: 'TEST_INPUT',
+  SETUP_ROUTING: 'SETUP_ROUTING',
+  CREATE_STREAM: 'CREATE_STREAM',
+  COMPLETE: 'COMPLETE',
+} as const;
+
+export type InputSetupWizardStep = typeof INPUT_WIZARD_STEPS[keyof typeof INPUT_WIZARD_STEPS]
+
+export type StepsData = {
+  [key in InputSetupWizardStep]?: any
```

Comment:
can we type this instead of any?

Reply:
Yes, but first we need the steps, for now this is just a skeleton.